models/sap.yml

```diff
@@ -1,11 +1,975 @@
 version: 2
 
 models:
+  - name: sap__0comp_code_attr
+    description: Company codes attributes
+    columns:
+      - name: bukrs
+        description: Company code
+      - name: waers
+        description: Currency key
+      - name: periv
+        description: Fiscal year variant
+      - name: ktopl
+        description: Chart of accounts
+      - name: bukrs
+        description: Company code
+      - name: land1
+        description: Country key
+      - name: kkber
+        description: Credit control area
+      - name: rcomp
+        description: Company
+
+  - name: sap__0comp_code_text
+    description: Company code text
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - mandt
+            - bukrs
+    columns: 
+      - name: mandt
+        description: Client
+      - name: bukrs
+        description: Company code
+      - name: txtmd
+        description:  Name of company code or company
+      - name: langu
+        description: Language key  
+
+  - name: sap__0company_text
+    description: Company text
```

Comment:
Request for more details in description of the model.

Reply:
Updated.